vm/src/import.rs

```diff
@@ -144,3 +144,35 @@ fn find_source(vm: &VirtualMachine, current_path: PathBuf, name: &str) -> Result
         None => Err(format!("No module named '{}'", name)),
     }
 }
+
+pub fn remove_importlib_frames(vm: &VirtualMachine, exc: &PyObjectRef) -> PyObjectRef {
```

Comment:
Are you sure you want to add this? This might be a good opportunity to keep the rustpython interpreter transparent to users, and show them what is actually going on. What is your opinion on this?

Reply:
I think that it gives too much information. I will add a `TODO` that when running on `verbose` this function will do nothing.